clang/test/Sema/alias-unused.cpp

```diff
@@ -0,0 +1,47 @@
+// RUN: %clang_cc1 -triple %itanium_abi_triple -Wunused -x c -verify %s
```

Comment:
consider using `git mv` when you rename files or move test cases. Makes it very obvious in code review when existing test cases are deleted vs moved and or updated.

Reply:
I did `git mv alias-unused.c alias-unused.cpp`, but git does not preserve file move history...

Once the tests are in a good shape, I will pre-commit the file and change the PR to show the diff.

Reply:
> but git does not preserve file move history...

Sure it does; isn't that the point of the `--follow` flag to `git log <file>`?

Reply:
Git does not track file moves explicitly. You'll find that `git log --follow` will work equally well whether you use `git add` and `git rm` or `git mv`. It uses a heuristic to decide that when a file disappears and a similar file appears it was probably moved.